ui-tests/tests/jupyter_scheduler.spec.ts

```diff
@@ -66,9 +73,9 @@ test.describe('Jupyter Scheduler', () => {
     await scheduler.createNotebook();
     await scheduler.createJobFromFilebrowser();
 
-    const timeStamp = scheduler.timestampLocator;
     await scheduler.assertSnapshot(FILENAMES.LIST_VIEW, {
-      mask: [timeStamp]
+      mask: [scheduler.timestamp],
+      maxDiffPixels: 420
```

Comment:
I would prefer us never using `maxDiffPixels`, as that kind of defeats the point of snapshot testing IMO. I have two questions:
1. What is causing the flakiness? What elements specifically are causing the layout to change test-to-test?

2. Is it possible to use a more narrow locator or a wider set of masks that doesn't cause any test flakiness?

Reply:
I agree that having `maxDiffPixels` is not sustainable and should be fixed in this or following PRs. 

Advanced Table causes it. All columns except job name shift left-right a couple of pixels between runs. Below is a heatmap of the difference. I have not caught this programmatically but I can reproduce a similar shift by editing content of the table cells and making it overflow. 


We could mask all columns in the table (headers and values) except of the job name or hardcode the values but IMO it also kinda defeats the purpose of the test.

![list-view-diff](https://github.com/jupyter-server/jupyter-scheduler/assets/26686070/6754eb36-bad7-428d-ab94-f0d3b2670c56)

Reply:
See if you can set the `create_time` timestamp retrieved from ListJobs and ListJobDefinitions to a fixed value via PlayWright: https://playwright.dev/docs/network#modify-responses

Reply:
I'd recommend a default of `0` (Jan 1, 1970 12:00 AM).

Reply:
Ok`page.route` was finicky but I made it work. Interestingly `0` parses to empty value and `1` parses to `Jan 1, 1970 12:00 AM)`
<img width="246" alt="Screenshot 2023-08-10 at 6 43 44 PM" src="https://github.com/jupyter-server/jupyter-scheduler/assets/26686070/cbf4bba9-86ac-4fc4-976c-2830c682c4c5">